.gitignore

```diff
@@ -1,7 +1,6 @@
 *.pyc
 *.log
 *.bak
-*.bat
```

Comment:
Feel free to update the file from the given source.

I would just like to remove *.bat files from here, both because they are old and IMHO can keep build instructions. It's like to ignore any *.sh files on real operating systems.